errors/error.go

```diff
@@ -138,6 +149,7 @@ func WithContext(err error, message string, fields Fields) error {
 	return NCError{
 		Causes:    []Cause{newCause, Cause{Message: err.Error()}},
 		Stack:     GetTrace(),
+		RawStack:  callers(),
```

Comment:
This way both: `GetTrace` and `callers` generate separate stack traces, which seems redundant. Can we refactor this, so that we generate the stack trace only once? For example by having a function that returns both stack & raw stack? (I wouldn't suggest to modify the `GetTrace` method since it is a public one and this would be a breaking change).